### PR TITLE
added armstrong.py implements armstrong number closes #4

### LINKED ISSUE #4: Armstrong number
Implement a function `is_armstrong` which checks if a number is armstrong using the following rule:

**Rule:** Return True if the sum of the cubes of the digits in the number is equal to the number itself, else return False. (You can also use this as your docstring)

Function signature:

```python

def is_armstrong(number): -> bool
    # TODO
    # return <True if the number is armstrong else False>
```

### PR DESCRIPTION
**Pull request by anu-coder**
@abhi8893 check the armstrong.py branch and close accept my pull request. This closes #4

### CODE REVIEW

File: scripts/armstrong.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# armstrong number: 153 = 1^3+5^3+3^3
+
+def is_armstrong(n):
+    s=0
+    a = n
+    while(n>0):
+        p = n %10
+        s = s + (p*p*p)
```
Comment (abhi8893) [inline scripts/armstrong.py]: Better to write it as `p**3` rather than `p*p*p`

File: scripts/armstrong.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# armstrong number: 153 = 1^3+5^3+3^3
+
+def is_armstrong(n):
+    s=0
```
Comment (abhi8893) [inline scripts/armstrong.py]: We can give it a more intuitive variable name such as `sum_of_cubes` or maybe just add a comment on the side as follows:

``` python
s = 0 # sum of cubes
````

File: scripts/armstrong.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        p = n %10
+        s = s + (p*p*p)
+        n = int(n/10)
+    if s == a:
+        return True
+    else:
+        return False
+
+n = int(input("Enter any number: "))
+print(is_armstrong(n))
```
Comment (abhi8893) [inline scripts/armstrong.py]: Put this as follows:

```python

if __name__ == '__main__':
    n = int(input("Enter any number: "))
    print(is_armstrong(n))

```
Follow this [link](https://www.freecodecamp.org/news/if-name-main-python-example/#:~:text=We%20can%20use%20an%20if,name%20if%20it%20is%20imported.) to learn why we would want to do this. 
But basically two reasons:
1. if you run it as ```python armstrong.py``` -> We want to take user input
2. if you want to import is_armstrong function -> We DON'T want to take user input, rather just import ```is_armstrong``` function as: 
```python 
from armstrong import is_armstrong
```

File: notebooks/.ipynb_checkpoints/Untitled-checkpoint.ipynb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+ "cells": [],
+ "metadata": {},
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```
Comment (abhi8893) [inline notebooks/.ipynb_checkpoints/Untitled-checkpoint.ipynb]: Add this file in ```.gitignore``` as we don't want each other's notebook checkpoint files.

File: .vscode/settings.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:\\Users\\Anurima\\AppData\\Local\\Continuum\\anaconda3\\python.exe"
+}
```
Comment (abhi8893) [inline .vscode/settings.json]: Add this file in ```.gitignore``` as we don't want each other's vscode's settings.json files.

**Review by abhi8893 — CHANGES_REQUESTED** [review]
I have suggested changes to ```armstrong.py``` + we need to remove ```.vscode``` and ``` .ipynb_checkpoints ``` folder by adding them to ```.gitignore```

File: .vscode/settings.json (hunk 5 above)
Comment (anu-coder) [reply .vscode/settings.json]: This I am not sure about but I have now assigned you to rectify if there is any more issue.

File: scripts/armstrong.py (hunk 3 above)
Comment (anu-coder) [reply scripts/armstrong.py]: Implemented

File: scripts/armstrong.py (hunk 2 above)
Comment (anu-coder) [reply scripts/armstrong.py]: Implemented

File: scripts/armstrong.py (hunk 1 above)
Comment (anu-coder) [reply scripts/armstrong.py]: Implemented

File: notebooks/.ipynb_checkpoints/Untitled-checkpoint.ipynb (hunk 4 above)
Comment (anu-coder) [reply notebooks/.ipynb_checkpoints/Untitled-checkpoint.ipynb]: implemented

**Pull request merged**